ui-particles-angular/projects/ui-particles-angular/src/lib/gio-breadcrumb/gio-breadcrumb.stories.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Meta, moduleMetadata } from '@storybook/angular';
+import { Story } from '@storybook/angular/types-7-0';
+import { withDesign } from 'storybook-addon-designs';
+import { MatButton, MatButtonModule } from '@angular/material/button';
+
+import { GioIconsModule } from '../gio-icons/gio-icons.module';
+
+import { GioBreadcrumbComponent } from './gio-breadcrumb.component';
+import { GioBreadcrumbModule } from './gio-breadcrumb.module';
+
+export default {
+  title: 'Components / Breadcrumb',
+  component: GioBreadcrumbComponent,
+  decorators: [
+    moduleMetadata({
+      imports: [GioBreadcrumbModule, MatButtonModule, GioIconsModule],
+    }),
+    withDesign,
+  ],
+  render: () => ({}),
+} as Meta;
+
+let route = 'plans';
+export const Default: Story = {
+  render: () => ({
+    template: `<gio-breadcrumb [breadcrumbItems]="breadcrumbItems"></gio-breadcrumb>`,
+    props: {
+      breadcrumbItems: [
+        {
+          targetRoute: '/apis',
+          displayName: 'Apis',
+        },
+        {
+          targetRoute: '/my-api',
+          displayName: 'My api',
+        },
+      ],
+      onClick: (target: string) => (route = target),
+      isActive: (target: string) => (route != target ? null : true),
+    },
+  }),
+};
+
+export const withComponent: Story = {
+  render: () => ({
+    template: `<gio-breadcrumb [breadcrumbItems]="breadcrumbItems"></gio-breadcrumb>`,
+    props: {
+      breadcrumbItems: [
+        {
+          targetRoute: '/apis',
+          displayName: 'Apis',
+        },
+        {
+          targetRoute: '/my-api',
+          component: MatButton,
```

Comment:
Sorry, 
But, I think it's not a good idea to allow adding components in this way